ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3875,4 +3944,20 @@ CSWTC:
   - OPERATIONAL
   uses:
   - condensing_supply_water_temperature_setpoint
-  - condensing_supply_water_temperature_sensor
\ No newline at end of file
+  - condensing_supply_water_temperature_sensor
+
+HTWHLSPC:
+  description: "Heat wheel control for AHUs"
```

Comment:
Heatwheel is an 'alternative' for ECON/mixed air units in AHUs. These are some of the points to expect from an AHU with a heatwheel

Reply:
You're right, I've moved the isolation damper controls to the AHU for now.

The heatwheel tries to maintain the supply setpoint so that should be in HTWHLSPC as well.

Reply:
If it is maintaining speed to supply air temperature, I would expect the following points:
```
  opt_uses:
  - heat_wheel_speed_percentage_sensor
  - return_air_temperature_sensor
  - exhaust_air_temperature_sensor
  uses:
  - heat_wheel_speed_percentage_command
  - heat_wheel_run_command
  - heat_wheel_run_status
  - outside_air_temperature_sensor
  - supply_air_temperature_sensor
  - supply_air_temperature_setpoint
```

This allows us to see if the heat wheel is effectively meeting its setpoint when its active.

Please adjust the points to these if it makes sense, and please make a few more changes:

- adjust the description to something like: "Heat wheel which controls supply temperature using speed control."
- rename it to something like HTWHLSTC (heat wheel supply temp control), so its clear its a subset of heat wheel control.
- please add `implements: CONTROL`

Reply:
That makes sense.

Will change.